bin.src/ataos_csc.py

```diff
@@ -5,10 +5,16 @@
 
 from lsst.ts.ataos import ataos_csc, version
 
-parser = argparse.ArgumentParser(f"Start the ATAOS CSC")
+parser = argparse.ArgumentParser("Start the ATAOS CSC")
 parser.add_argument("--version", action="version", version=version.__version__)
-parser.add_argument("-v", "--verbose", dest="verbose", action='count', default=0,
-                    help="Set the verbosity for console logging.")
+parser.add_argument(
+    "-v",
+    "--verbose",
+    dest="verbose",
+    action="count",
+    default=0,
+    help="Set the verbosity for console logging.",
+)
```

Comment:
Is this argument used for anything? I think it shows up as .verbose in the parsed args, but is then ignored. If so, please remove it.